.github/workflows/pull-request.yaml

```diff
@@ -21,6 +21,8 @@ jobs:
       tox-version: ""
       python-version: "3.10"
 
-  check-file-updates:
+  sync-from-sunbeam:
     name: Check updates for charm files
-    uses: ./.github/workflows/_check_file_update.yaml
+    uses: ./.github/workflows/sync-from-sunbeam.yaml
```

Comment:
I don't think we should call this workflow from the pull request checks workflow.  The sync workflow opens a pull request to make updates; it doesn't do any checks or report on status, so it doesn't add information to a pull request.